src/m365/spo/commands/site/site-remove.ts

```diff
@@ -114,6 +99,45 @@ class SpoSiteRemoveCommand extends SpoCommand {
     }
   }
 
+  private deleteUnGroupifiedSite(logger: Logger, args: CommandArgs): Promise<void> {    
```

Comment:
While you often heard about _groupified_ sites, I haven't heard of _ungroupified_ sites. Let's call them _sites without groups_ instead to avoid inventing new words ;)

Reply:
Thanks for that suggestion👍.Even I was in a dilemma whether I should do that or not. 
Totally agree, already we have enough terminologies, let us not add more.💪💪